src/sentry/snuba/referrer.py

```diff
@@ -160,13 +150,19 @@ class Referrer(Enum):
     )
     API_ORGANIZATION_EVENTS_V2 = "api.organization-events-v2"
     API_ORGANIZATION_EVENTS = "api.organization-events"
+    API_ORGANIZATION_METRICS_DATA = "api.organization.metrics-data"
     API_ORGANIZATION_METRICS_ESTIMATION_STATS = "api.organization-metrics-estimation-stats"
+    API_ORGANIZATION_METRICS_METADATA_FETCH_SPANS = "api.organization.metrics-metadata.fetch-spans"
+    API_ORGANIZATION_METRICS_METADATA_FETCH_METRICS_SUMMARIES = (
```

Comment:
Called both of theme `metadata` since in the future I would like to remove `/ddm/meta` to a new endpoint called `/metrics/metadata`